### PR TITLE
Add stylelint-prettier plugin

### PR DESCRIPTION
**Pull request by BPScott**
> Which issue, if any, is this issue related to?

None, as it's a documentation fix.

> Is there anything in the PR that needs further explanation?

No

### CODE REVIEW

**Comment by BPScott** [comment]
Hi @ntwb,

`prettier-stylelint` is a wrapper that runs prettier then stylelint. This means instead of running `stylelint`, you run a new executable `prettier-stylelint`. This is useful if you want stylelint to make adjustments to prettier's output, but it's a new command for people to remember. This is good for autofixing but makes identifying if changes need to be made without making corrections a little tricky. 

`stylelint-prettier` is a plugin to stylelint that exposes prettier modifications as a stylelint rule. This means you continue to run `stylelint` as you would before and prettier changes are exposed as errors in stylelint's reporting. This means people don't need to change their existing process - running `stylelint` shall report on changes and `stylelint --fix` shall fix them.

File: docs/user-guide/plugins.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Plugins are rules and sets of rules built by the community that support methodol
 -   [`stylelint-no-indistinguishable-colors`](https://github.com/ierhyna/stylelint-no-indistinguishable-colors): Disallow colors that are suspiciously close to being identical.
 -   [`stylelint-no-unsupported-browser-features`](https://github.com/ismay/stylelint-no-unsupported-browser-features): Disallow features that are unsupported by the browsers that you are targeting.
 -   [`stylelint-order`](https://github.com/hudochenkov/stylelint-order): Specify the ordering of things e.g. properties within declaration blocks (plugin pack).
+-   [`stylelint-prettier`](https://github.com/BPScott/stylelint-prettier): Runs [Prettier](https://prettier.io/) as a stylelint rule
```
Comment (hudochenkov) [inline docs/user-guide/plugins.md]: Could you put a dot at the end, please?

**Comment by ntwb** [comment]
Thanks for the detailed description @BPScott 💯

**Comment by jeddy3** [comment]
I think this could be one of the plugins we consider highlighting when we come to do https://github.com/stylelint/stylelint/issues/3362.

@BPScott Thanks for the good work and the plugin!

**Pull request merged**